python/chemiscope/jupyter.py

```diff
@@ -282,6 +285,9 @@ def show_input(path, *, mode="default", warning_timeout=10000, cache_structures=
     except KeyError:
         raise ValueError("missing metadata in chemiscope.load")
 
+    if settings is not None:
+        dict_input["settings"].update(settings)
```

Comment:
maybe we can add some validations like:
```suggestion
    if settings is not None:
        if not isinstance(settings, dict):
            raise ValueError(f"expected 'settings' to be a dict, got {type(settings)} instead")
        if "settings" not in dict_input:
            dict_input["settings"] = {}
        dict_input["settings"].update(settings)
```
(linter might fall if suggestion is applied)